# This file configures the DepthImageOctomapUpdater. It's faster than the PointCloudOctomapUpdater, but it doesn't work in Gazebo (because the simulated depth image is broken).

sensors:
  - sensor_plugin: occupancy_map_monitor/DepthImageOctomapUpdater
    image_topic: /gripper_cam/depth_registered/image_raw
    far_clipping_plane_distance: 1.0
    near_clipping_plane_distance: 0.25
    padding_scale: 1.0
    padding_offset: 0.1
    queue_size: 5
    skip_vertical_pixels: 1
    skip_horizontal_pixels: 1
    shadow_threshold: 0.2
#   filtered_cloud_topic: filtered_cloud
# this last topic is just to debug, having it adds processing time
